include/game/parameters/characters.h

```diff
@@ -0,0 +1,20 @@
+#ifndef GUARD_PARAMETERS_CHARACTERS_H
+#define GUARD_PARAMETERS_CHARACTERS_H
+
+/*** Common ***/
+#define PLAYER_GRAVITY (0.033) // = (8. / 256.)
+
+/*** Tricks ***/
+#define TRICK__STOP_N_SLAM__DROP_SPEED Q_24_8(0.21875)
+
+/*** Cream ***/
+#define CREAM_FLYING_DURATION (4 * GBA_FRAMES_PER_SECOND)
+
+/*** Tails ***/
+// NOTE: Unlike Cream, Tails does fly for 8 seconds, but his initial value gets set to 4.
+//       For some reason they lower his timer only every 2nd frame (in PlayerCB_8012C2C),
+//       instead of using a bigger value from the get-go, even though they gave him a
+//       4-byte timer.
+#define TAILS_FLYING_DURATION ((8 * GBA_FRAMES_PER_SECOND) / 2)
```

Comment:
In this case, could make a `TAILS_FLIGHT_MULTIPLIER 2` and do `... & (TAILS_FLIGHT_MULTIPLIER - 1)`

Reply:
I mean... it's not really a multiplier, I guess?
It's just skipping every other frame...

Maybe with a different name?